portal-ui/src/screens/LoginPage/LoginPage.tsx

```diff
@@ -578,7 +578,7 @@ const Login = () => {
           <Grid item xs={12} className={classes.linkHolder}>
             <div className={classes.miniLinks}>
               <a
-                href="https://docs.min.io/?ref=con"
+                href="https://min.io/docs/minio/kubernetes/upstream/index.html?ref=con"
```

Comment:
I think this should lead to  linux page instead of k8s one
```suggestion
                href="https://min.io/docs/minio/linux/index.html"
```